apps/meteor/server/models/raw/Messages.ts

```diff
@@ -389,6 +390,10 @@ export class MessagesRaw extends BaseRaw<IMessage> implements IMessagesModel {
 			query.t = { $nin: types };
 		}
 
+		if (!showSystemMessages) {
```

Comment:
Shouldn't we validate this against `types`? I mean, it wouldn't make sense to have both at the same time, and if so, maybe we should guarantee that.

Reply:
This was done intentionally, explained in different comment why.